jep/0000/README.adoc

```diff
@@ -0,0 +1,366 @@
+Ending support for Java 8 allows Jenkins core and Jenkins plugins to use libraries that support Java 11 but do not support Java 8.
+
+== Reasoning
+
+“Goals and non-goals” section in the specification lists design decisions taken
+to ensure it can be delivered by a small team.
+Non-goals in the specification are defined to limit the scope of work.
+The main objective is to move Jenkins core development to Java 11.
+There will be follow-up tasks for further improvements and to adopt new features.
+
+=== Support of Java 17
+
+This JEP intentionally limits its scope by not including Java 17 support.
+It does not prevent work on Java 17, but that work is outside the scope of this JEP.
+
+=== Docker image labeling
+
+Docker image labels were updated in August 2021 to use link:https://www.jenkins.io/blog/2021/08/17/docker-images-use-jdk-11-by-default/[Java 11 by default].
+The image labels that do not explicitly mention a Java version (like `latest`, `lts`, `slim`, `alpine`) are already delivering Java 11.
+
+Image labels that explicitly mention `jdk8` will not be updated after Jenkins core ends support for Java 8.
+
+== Backwards Compatibility
+
+The following backward compatibility requirements are defined:
+
+* Jenkins core and updated plugins should fully support Java 11
+* Jenkins plugins may continue to compile with Java 8 so long as the plugins run successfully with Java 11
+* Jenknis plugins that require a Jenkins version that does not support Java 8 will be expected to compile with Java 11
+
+== Security
+
+=== Process
+
+Only Java 11 with the latest security fixes will be supported at the moment of the public release.
+
+Jenkins security issues on the release that ends support of Java 8 will be processed according to the
+standard link:https://jenkins.io/security/[Jenkins Security Process].
+
+=== Security risks
+
+* No additional security risks are expected due to Jenkins ending support for Java 8
+
+== Infrastructure Requirements
+
+=== ci.jenkins.io
+
+* Tool Infrastructure should continue to offer the latest version of Java 11
+
+=== Jenkins Pipeline Library
+
+* `buildPlugin()`, `runATH()`, and `runPCT()` will run tests with JDK 11
+
+=== DockerHub
+
+* Dockerhub will continue to host container images for Java 11
+
+== Testing
+
+Ending Java 8 support in Jenkins requires significant testing.
+Community contributors will be encouraged to test environments and configurations to assure that Jenkins core no longer requires Java 8.
+
+A link:https://docs.google.com/document/d/13ttjJ7HaUkYMy3L5P8D7w7TddqrUr-1IojtZCukFBQ8/edit?usp=sharing[status reporting document] is ready to track the testing effort.
+Testers are welcome to report their results there.
+
+Tests to be performed:
+
+* ATH is updated as needed to run on Java 11
+* ATH is performed successfully on Java 11
+* PCT is updated as needed to run on Java 11
+* PCT is performed successfully on Java 11
+* Packaging tests are performed successfully on Java 11
```

Comment:
I think @dduportal might have already done this

Reply:
Not yet (ref. https://github.com/jenkins-infra/release/issues/199) but JDK11 is available on the current release process so easy to switch.
We can plan to do it for the 1st time for the weekly of the 7th of feb.(or later).